templates/hardwarelab/event_manager.html

```diff
@@ -0,0 +1,198 @@
+{% extends "layout.html" %}
+{% block content %}
+<script>
+$(document).ready(function(){
+    $(document).on("click", ".create-rental", function() {
+      $rental_id = $(this).siblings("#rental-id").attr("value");
+      $inventory_id = $(this).parentsUntil('#reservations').find('select').val();
+      console.log($inventory_id);
+      if($inventory_id == ''){
+        console.log('empty');
+        alertify.alert("Please select an Inventory ID");
+      } else {
+        $("#create-rental-modal").modal({
+          closable  : true,
+          onDeny    : function(){
+            return true;
+          },
+          onApprove : function() {
+            $.ajax({
+              url : "/api/rent/", // the endpoint
+              type : "GET", // http method
+              data : { 
+                rental_id: $rental_id,
+                inventory_id: $inventory_id
+              },
+              success : function(data) {
+                $("#manager-container").load(document.URL + ' #manager-container');
+                // $("#manager-container").replaceWith(data);
```

Comment:
if subsequent attempts fail, there must be a bug in the backend code. I'll check later what's up. 

The reason it's slow is because you're doing one ajax call which returning the page and then calling again for a .get request.